.github/workflows/close-issues.yml

```diff
@@ -0,0 +1,22 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 30 days with no activity."
```

Comment:
Maybe add something like:

"... It will close in 14 days, if there's no activity, or if the `stale` label is not removed. Does anyone want to add something?"

Reply:
That is a very good remark! I've updated the string in bb5f6df871dee17201d0bce89ac2b769b6be0ef9